sources/nidaba/graphemes.tsv

```diff
@@ -1,36 +1,25 @@
-(?)	d̠͡ɮ̠ʰː	dental:affricate:+voice +lateral +spread glottis	\t{\textsubbar{d}͡\textsubbar{\textlyoghlig}} extsuperscript{h}\textlengthmark
-(?)	d̠͡ɮ̠ʲ	dental, palatal:affricate:+voice +lateral	\t{\textsubbar{d}͡\textsubbar{\textlyoghlig}}\super j
-(?)	d̠͡ɮ̠ʲː	dental, palatal:affricate:+voice +lateral	\t{\textsubbar{d}͡\textsubbar{\textlyoghlig}}\super j\textlengthmark
-(?)	d̠͡ɮ̠ʷ	dental, labial:affricate:+voice +lateral	\t{\textsubbar{d}͡\textsubbar{\textlyoghlig}}\super w
-(?)	d̠͡ɮ̠ʷː	dental, labial:affricate:+voice +lateral	\t{\textsubbar{d}͡\textsubbar{\textlyoghlig}}\super w\textlengthmark
-(?)	d̠͡ɮ̠ː	dental:affricate:+voice +lateral	\t{\textsubbar{d}͡\textsubbar{\textlyoghlig}}\textlengthmark
-(?)	d̪͡ɮ̪	dental:affricate:+voice +lateral	\t{\textsubbridge{d}͡\textsubbridge{\textlyoghlig}}
-(?)	d̪͡ɮ̪ʰ	dental:affricate:+voice +lateral +spread glottis	\t{\textsubbridge{d}͡\textsubbridge{\textlyoghlig}} extsuperscript{h}
-(?)	d̪͡ɮ̪ʰː	dental:affricate:+voice +lateral +spread glottis	\t{\textsubbridge{d}͡\textsubbridge{\textlyoghlig}} extsuperscript{h}\textlengthmark
-(?)	d̪͡ɮ̪ʲ	dental, palatal:affricate:+voice +lateral	\t{\textsubbridge{d}͡\textsubbridge{\textlyoghlig}}\super j
-(?)	d̪͡ɮ̪ʲː	dental, palatal:affricate:+voice +lateral	\t{\textsubbridge{d}͡\textsubbridge{\textlyoghlig}}\super j\textlengthmark
-(?)	d̪͡ɮ̪ʷ	dental, labial:affricate:+voice +lateral	\t{\textsubbridge{d}͡\textsubbridge{\textlyoghlig}}\super w
-(?)	d̪͡ɮ̪ʷː	dental, labial:affricate:+voice +lateral	\t{\textsubbridge{d}͡\textsubbridge{\textlyoghlig}}\super w\textlengthmark
-(?)	d̪͡ɮ̪ː	dental:affricate:+voice +lateral	\t{\textsubbridge{d}͡\textsubbridge{\textlyoghlig}}\textlengthmark
-(?)	ɺ̪	dental:liquid:+voice +lateral	\textsubbridge{\textturnlonglegr}
-(?)	ɺ̪ʲ	dental, palatal:liquid:+voice +lateral	\textsubbridge{\textturnlonglegr}\super j
+ɳ̥	ɳ̥	retroflex:nasal:−voice +nasal	*\:n
+ɭ̊	ɭ̥	retroflex:liquid:−voice +lateral	*\textturny
+ɲ̍	ɲ̩	palatal:nasal:+voice +nasal +syllabic	\textipa{\s{\textltailn}}
+ɳ̍	ɳ̩	retroflex:nasal:+voice +nasal +syllabic	\textipa{\s{\:n}}
 <NA>		empty:empty:	
-<NA>	t͡ɬ̲	alveolar:affricate:+voice +lateral	\t{t͡\textsubbar{\textbeltl}}
-<NA>	t͡ɬ̲ʰ	alveolar:affricate:+voice +lateral +spread glottis	\t{t͡\textsubbar{\textbeltl}} extsuperscript{h}
-<NA>	t͡ɬ̲ʰː	alveolar:affricate:+voice +lateral +spread glottis	\t{t͡\textsubbar{\textbeltl}} extsuperscript{h}\textlengthmark
-<NA>	t͡ɬ̲ʲ	alveolar, palatal:affricate:+voice +lateral	\t{t͡\textsubbar{\textbeltl}}\super j
-<NA>	t͡ɬ̲ʲː	alveolar, palatal:affricate:+voice +lateral	\t{t͡\textsubbar{\textbeltl}}\super j\textlengthmark
-<NA>	t͡ɬ̲ʷ	alveolar, labial:affricate:+voice +lateral	\t{t͡\textsubbar{\textbeltl}}\super w
-<NA>	t͡ɬ̲ʷː	alveolar, labial:affricate:+voice +lateral	\t{t͡\textsubbar{\textbeltl}}\super w\textlengthmark
-<NA>	t͡ɬ̲ː	alveolar:affricate:+voice +lateral	\t{t͡\textsubbar{\textbeltl}}\textlengthmark
+<NA>	d̠͡ɮ̠	dental:affricate:+voice +lateral	\t{\textsubbar{d}͡\textsubbar{\textlyoghlig}}
```

Comment:
I would be inclined to just map these to dɮ etc. as you have done with the voiceless ones below

Reply:
Makes sense, since this mapping is regular. Teh data was automatically produced by the author by combining features.